### PR TITLE
Fix deprecated link to plugin (#1109)

### LINKED ISSUE #1109: GraphiQL is not rendering (Explorer plugin error)
## Problem

There is a deprecated link to download GraphiQL explorer plugin in [the GraphiQL template](https://github.com/mirumee/ariadne/blob/d94a05eef136a333d66cb81ff650eedabbca9a4d/ariadne/explorer/templates/graphiql.html#L66). It results in a 404 error that unable to render the GraphiQL Playground when Explorer plugin is activated.

image

## Solution

Replace `graphiql-plugin-explorer.umd.js` file for `index.umd.js` file.

### PR DESCRIPTION
**Pull request by juansedo**
The issue #1109 explains the context about this change.

The explorer plugin file has been renamed and this is generating a 404 error for those projects that are using Ariadne with GraphiQL and the plugin enabled.

Fixes #1109

### CODE REVIEW

**Review by rafalp — APPROVED** [review]

**Comment by rafalp** [comment]
Please fix tests failing.

**Review by rafalp — CHANGES_REQUESTED** [review]
Tests failing

File: ariadne/explorer/templates/graphiql.html [hunk 1]
```diff
@@ -63,7 +63,7 @@
     {% if enable_explorer_plugin %}
     <script
       crossorigin
-      src="https://unpkg.com/@graphiql/plugin-explorer/dist/graphiql-plugin-explorer.umd.js"
+      src="https://unpkg.com/@graphiql/plugin-explorer/dist/index.umd.js"
```
[the remark this answers is not in the document]
Comment (rafalp) [reply ariadne/explorer/templates/graphiql.html:66]: We would have to pin both the plugin and GraphiQL IMHO.

**Comment by juansedo** [comment]
@rafalp, any recommendation (or command snippet) to regenerate snapshots?

**Comment by rafalp** [comment]
You have this recommendation in the test runner itself ;)

> 6 snapshots failed in 3 test suites. Inspect your code or run with `pytest --snapshot-update` to update them.

**Review by rafalp — APPROVED** [review]

**Comment by rafalp** [comment]
Thanks. I'll release 0.20.1 with a fix tomorrow.

**Pull request merged**